### PR TITLE
Change StaticRNN to fluid.layers.rnn

### PR DESCRIPTION
**Pull request by lfchener**
Change StaticRNN to fluid.layers.rnn.

### CODE REVIEW

File: model_utils/network.py [hunk 1]
```diff
@@ -137,20 +140,32 @@ def bidirectional_simple_rnn_bn_layer(name, input, size, share_weights):
...
+            cell=forward_cell, inputs=input, time_major=False, is_reverse=False)
+        forward_rnn = fluid.layers.sequence_unpad(x=forward_rnn, length=length)
+
+        reverse_cell = RNNCell(
+            hidden_size=size,
+            activation=fluid.layers.brelu,
             param_attr=fluid.ParamAttr(name=name + '_reverse_rnn_weight'),
-            bias_attr=fluid.ParamAttr(name=name + '_reverse_rnn_bias'),
+            bias_attr=fluid.ParamAttr(name=name + '_reverse_rnn_bias'))
+        input, length = fluid.layers.sequence_pad(input_proj_bn, pad_value)
```
Comment (kuke) [inline model_utils/network.py]: This line is redundant for we have line 151

File: model_utils/network.py [hunk 2]
```diff
@@ -183,18 +198,32 @@ def bidirectional_simple_rnn_bn_layer(name, input, size, share_weights):
...
+            activation=fluid.layers.brelu,
             param_attr=fluid.ParamAttr(name=name + '_forward_rnn_weight'),
-            bias_attr=fluid.ParamAttr(name=name + '_forward_rnn_bias'),
-            is_reverse=False)
-        reverse_rnn = simple_rnn(
-            input=input_proj_bn_backward,
-            size=size,
+            bias_attr=fluid.ParamAttr(name=name + '_forward_rnn_bias'))
+
+        pad_value = fluid.layers.assign(input=np.array([0.0], dtype=np.float32))
```
Comment (kuke) [inline model_utils/network.py]: Not necessary

File: model_utils/network.py [hunk 3]
```diff
@@ -183,18 +198,32 @@ def bidirectional_simple_rnn_bn_layer(name, input, size, share_weights):
...
+        input, length = fluid.layers.sequence_pad(input_proj_bn, pad_value)
```
Comment (kuke) [inline model_utils/network.py]: `input_proj_bn` is not defined in `else` branch, please remember to test all the branches before pull request. And also please try to reuse these lines with the `if` branch.

File: model_utils/network.py [hunk 4]
```diff
@@ -59,50 +59,53 @@ def conv_bn_layer(input, filter_size, num_channels_in, num_channels_out, stride,
...
 
-def simple_rnn(input, size, param_attr=None, bias_attr=None, is_reverse=False):
-    '''A simple rnn layer.
-    :param input: input layer.
-    :type input: Variable
-    :param size: Dimension of RNN cells.
-    :type size: int
+class RNNCell(fluid.layers.RNNCell):
+    '''A simple rnn cell.
+    :param hidden_size: Dimension of RNN cells.
```
[the remark this answers is not in the document]
Comment (lfchener) [reply model_utils/network.py]: Done

File: model_utils/network.py (hunk 1 above)
Comment (lfchener) [reply model_utils/network.py]: This line has been deleted

File: model_utils/network.py (hunk 3 above)
Comment (lfchener) [reply model_utils/network.py]: Done

File: model_utils/network.py [hunk 5]
```diff
@@ -137,20 +140,32 @@ def bidirectional_simple_rnn_bn_layer(name, input, size, share_weights):
...
+            activation=fluid.layers.brelu,
             param_attr=fluid.ParamAttr(name=name + '_forward_rnn_weight'),
-            bias_attr=fluid.ParamAttr(name=name + '_forward_rnn_bias'),
-            is_reverse=False)
+            bias_attr=fluid.ParamAttr(name=name + '_forward_rnn_bias'))
 
-        reverse_rnn = simple_rnn(
-            input=input_proj_bn,
-            size=size,
+        pad_value = fluid.layers.assign(input=np.array([0.0], dtype=np.float32))
```
[the remark this answers is not in the document]
Comment (lfchener) [reply model_utils/network.py]: Sequence_pad must set pad_value.

File: model_utils/network.py (hunk 2 above)
Comment (lfchener) [reply model_utils/network.py]: pad_value must be set

**Pull request merged**